dts/arm/nuvoton/npcx/npcx4.dtsi

```diff
@@ -63,6 +63,13 @@
 	soc {
 		compatible = "nuvoton,npcx4", "nuvoton,npcx", "simple-bus";
 
+		bbram: bb-ram@400af001 {
+			compatible = "nuvoton,npcx-bbram";
+			reg = <0x400af001 0x7F
```

Comment:
Add a comment here describing the errata.

Reply:
Done.